Xamarin.PropertyEditing/ValueInfo.cs

```diff
@@ -15,7 +12,7 @@ public T Value
 		}
 
 		/// <summary>
-		/// Gets or sets a descriptor of the value source, such as a resource reference or binding description.
+		/// Gets or sets a descriptor of the value, such as a string value name or <see cref="ITypeInfo"/>.
```

Comment:
This is actually difficult and subtle. For developers integrating the property panel, adding more details about the difference between value and source descriptors would be very useful.

Reply:
What more can I add than the exact types you might expect in each property like I did?

Reply:
Two things I think: first, some explanation about why they are separated (we had to debate whether they should, which indicates it's not a trivial thing), and also maybe more details about how you'd implement it. When integrating this, you'll have to set those values, but you'll also have to use them, unless I misunderstood. Maybe walking through the top-level steps of a use case would be useful.

Reply:
Discussion details are in the commit expanded message. We didn't really debate whether they should be separated, just how to separate them, the need for separation was obvious.

I can add a seealso targeted at `IObjectEditor.SetValueAsync`, implementation details are provided there including examples of this property. I'll also see if I need to add more detail there.